tests/Config/ConfigurationLoaderTest.php

```diff
@@ -145,131 +147,135 @@ function (?string $eTag) use ($flagsResourceResponse, $flagsRaw) {
-//                }
-//            }
-//        }';
-//
-//        $warmBanditsRaw = '{
-//            "bandits": {
-//                "cold_starting_bandit" : {
-//                    "banditKey": "cold_starting_bandit",
-//                    "modelName": "falcon",
-//                    "updatedAt": "2023-09-13T04:52:06.462Z",
-//                    "modelVersion": "v1",
-//                    "modelData": {
-//                        "gamma": 1.0,
-//                        "defaultActionScore": 0.0,
-//                        "actionProbabilityFloor": 0.0,
-//                        "coefficients": {}
-//                    }
-//                }
-//            }
-//        }';
-//
-//
-//        $apiWrapper = $this->getMockBuilder(APIRequestWrapper::class)->disableOriginalConstructor()->getMock();
-//
-//        $apiWrapper->expects($this->exactly(3))
-//            ->method('getUFC')
-//            ->willReturnOnConsecutiveCalls(
-//                new APIResource($initialFlagsRaw, true, "initial"),
-//                new APIResource($initialFlagsRaw, true, "initialButForced"),
-//                new APIResource($warmFlagsRaw, true, "warm"),
-//            );
-//
-//        $apiWrapper->expects($this->exactly(2))
-//            ->method('getBandits')
-//            ->willReturnOnConsecutiveCalls(
-//                new APIResource($coldBanditsRaw, true, null),
-//                new APIResource($warmBanditsRaw, true, null),
-//            );
-//
-//        $configStore = new ConfigStore(DefaultCacheFactory::create());
-//        $loader = new ConfigurationLoader($apiWrapper, $configStore);
-//
-//
-//        // First fetch has the bandit cold
-//        $loader->fetchAndStoreConfiguration(null);
-//
-//        $bandit = $loader->getBandit('cold_starting_bandit');
-//        $this->assertNotNull($bandit);
-//        $this->assertInstanceOf(Bandit::class, $bandit);
-//        $this->assertEquals('cold_starting_bandit', $bandit->banditKey);
-//        $this->assertEquals('cold start', $bandit->modelVersion);
-//
-//
-//        // Trigger a reload, second fetch shows the bandit as still cold
-//        $loader->fetchAndStoreConfiguration('initial');
-//
-//        $bandit = $loader->getBandit('cold_starting_bandit');
-//        $this->assertNotNull($bandit);
-//        $this->assertInstanceOf(Bandit::class, $bandit);
-//        $this->assertEquals('cold_starting_bandit', $bandit->banditKey);
-//        $this->assertEquals('cold start', $bandit->modelVersion);
-//
-//        // Trigger a reload, third fetch has the bandit warm with v1
-//        $loader->fetchAndStoreConfiguration('initialButForced');
-//
-//        $bandit = $loader->getBandit('cold_starting_bandit');
-//        $this->assertNotNull($bandit);
-//        $this->assertInstanceOf(Bandit::class, $bandit);
-//        $this->assertEquals('cold_starting_bandit', $bandit->banditKey);
-//        $this->assertEquals('v1', $bandit->modelVersion);
-//    }
+    /**
```

Comment:
Except for here, where we restore the old test case

Reply:
💪